sdk/python/feast/infra/registry/remote.py

```diff
@@ -59,6 +60,11 @@ class RemoteRegistryConfig(RegistryConfig):
     """ str: Path to the public certificate when the registry server starts in TLS(SSL) mode. This may be needed if the registry server started with a self-signed certificate, typically this file ends with `*.crt`, `*.cer`, or `*.pem`.
     If registry_type is 'remote', then this configuration is needed to connect to remote registry server in TLS mode. If the remote registry started in non-tls mode then this configuration is not needed."""
 
+    is_tls: bool = False
+    """ bool: if you are planning to connect the registry server which started in TLS(SSL) mode then this should be true.
```

Comment:
```suggestion
    """     bool: Set to `True` if you plan to connect to a registry server running in TLS (SSL) mode. 
    If you intend to add the public certificate to the trust store instead of passing it via the `cert` parameter, 
    this field must be set to `True`.
```